src/examples/meshes.zig

```diff
@@ -96,8 +99,16 @@ fn on_init() !void {
         .texture_1 = tex_emissive,
     });
 
+    materials = std.ArrayList(graphics.Material).init(delve.mem.getAllocator());
+    try materials.append(material);
+
+    const path = "assets/meshes/";
+    const filename = "SciFiHelmet.gltf";
+
+    gltf_data = try gltf.loadData(filename, path);
+
     // Load our mesh!
-    mesh_test = mesh.Mesh.initFromFile(delve.mem.getAllocator(), "assets/meshes/SciFiHelmet.gltf", .{ .material = material });
+    mesh_test = mesh.Mesh.initFromData(delve.mem.getAllocator(), gltf_data, 0, .{ .materials = materials });
```

Comment:
This change from 'initFromFile' to 'initFromData' is making this API much harder to use for everyone that wants to load a mesh. My thinking is that the GLTF loading should be abstracted, the end user of the API should not need to worry about the specifics.

Reply:
I see the point you are making. Before this change delve assumed that a GLTF file contains a single mesh with a single material and that is not the case. 

Should we have another API like Model that has a initFromFile and behind the scenes uses a lower level Mesh?